[3.0.0-alpha.0 upgrade] can't resolve react-testing-library

Thank you very much for this alpha release i think it will be very helpful

after upgrading 
react-admin => 3.0.0-alpha.0
@material-ui/core => 4.2.1
@material-ui/icons => 4.2.1

i got this error

_ra-core/esm/util/renderWithRedux.js
Module not found: Can't resolve 'react-testing-library'_

i tried to install it but i got another error 

_Error: 🚨  react-testing-library has moved to @testing-library/react. Please uninstall react-testing-library and install @testing-library/react instead, or use an older version of react-testing-library. Learn more about this change here: https://github.com/testing-library/dom-testing-library/issues/260 Thanks! :)_

--- a/packages/ra-core/package.json
+++ b/packages/ra-core/package.json
@@ -42,7 +42,6 @@
         "react": "~16.8.0",
         "react-dom": "~16.8.0",
         "react-test-renderer": "~16.8.6",
-        "react-testing-library": "^7.0.0",
         "rimraf": "^2.6.3"
     },
     "peerDependencies": {
@@ -62,6 +61,7 @@
         "react-redux": "^7.1.0",
         "react-router": "^5.0.1",
         "react-router-dom": "^5.0.1",
+        "react-testing-library": "^7.0.0",
         "recompose": "~0.26.0",
         "redux": "^3.7.2 || ^4.0.0",
         "redux-form": "~8.2.0",

